Lib/tkinter/tix.py

```diff
@@ -63,6 +65,9 @@
 TCL_IDLE_EVENTS   = 1 << 5
 TCL_ALL_EVENTS    = 0
 
+warnings.warn('The Tix Tk extension is unmaintained, and the tkinter.tix wrapper is deprecated in '
```

Comment:
Maximum line length for the standard library is 79 characters, please wrap this one.